Backend/app/rider-platform/rider-app/Main/spec/Storage/ticket.yaml

```diff
@@ -225,6 +261,8 @@ TicketBooking:
     status: BookingStatus
     createdAt: UTCTime
     updatedAt: UTCTime
+    bookedSeats: Int
```

Comment:
Why this is required here?

Reply:
we have to update the status of ticket booking if all seats are cancel then make it cancelled so for doing that we need to do either dfs search or we would have made flag if bookedSeats == cancelledSeats then mark it cancelled